.github/workflows/release.yaml

```diff
@@ -0,0 +1,69 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to be released'
+        required: true
+
+permissions:
+  contents: write
+  packages: write
+
+jobs:
+  check:
+    name: preliminary-checks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Ensure version is properly set
+        run: |
+          test "$(npm -s run get-version)" = "${{ inputs.tag }}"
```

Comment:
Facing a bit of a dilema over this one, so on one hand we can make sure we set the version right, manually each time (like we already do today). Or on the other hand, we could add a command like this before publishing the packages:

```shell
yarn version --new-version ${{ inputs.tag }} --no-git-tag-version
```

Which will set the right `version` in the `package.json`, and we can leave some kind of placeholder there that we never touch.

If anyone has any opinion on this one, would love to hear it.